contrib/openssl-cmake/CMakeLists.txt

```diff
@@ -1447,6 +1415,11 @@ set(SSL_SRC
     ${OPENSSL_SOURCE_DIR}/ssl/statem/statem_srvr.c
 )
 
+# Disable all deprecated API
+add_definitions(-DOPENSSL_NO_DEPRECATED)
+add_definitions(-DOPENSSL_NO_ENGINE)
+add_definitions(-DOPENSSL_NO_SRP)
```

Comment:
Wondering why SRP is disabled? Couldn't find anything on https://en.wikipedia.org/wiki/Secure_Remote_Password_protocol that says it is insecure or outdated.

Reply:
https://docs.openssl.org/3.1/man3/SRP_Calc_B/#history